_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -220,3 +218,336 @@ Distros: openshift-rosa-hcp
+    File: nodes-cma-autoscaling-custom-removing
+- Name: Controlling pod placement onto nodes (scheduling)
+  Dir: scheduling
+  Topics:
+  - Name: About pod placement using the scheduler
+    File: nodes-scheduler-about
+  - Name: Placing pods relative to other pods using pod affinity and anti-affinity rules
+    File: nodes-scheduler-pod-affinity
+  - Name: Controlling pod placement on nodes using node affinity rules
+    File: nodes-scheduler-node-affinity
+  - Name: Placing pods onto overcommited nodes
+    File: nodes-scheduler-overcommit
+# Per OSDOCS-9791, ROSA customers cannot add taints to individual nodes.
+#  - Name: Controlling pod placement using node taints
+#    File: nodes-scheduler-taints-tolerations
+  - Name: Placing pods on specific nodes using node selectors
+    File: nodes-scheduler-node-selectors
+  - Name: Controlling pod placement using pod topology spread constraints
+    File: nodes-scheduler-pod-topology-spread-constraints
+# - Name: Placing a pod on a specific node by name
+#   File: nodes-scheduler-node-names
+# - Name: Placing a pod in a specific project
+#   File: nodes-scheduler-node-projects
+# - Name: Keeping your cluster balanced using the descheduler
+#   File: nodes-scheduler-descheduler
+# Cannot create namespace to install Desceduler Operator; revisit after Operator book converted
+#  - Name: Evicting pods using the descheduler
+#   File: nodes-descheduler
+# Cannot create namespace to install Secondary Scheduler Operator; revisit after Operator book converted
+# - Name: Secondary scheduler
+#   Dir: secondary_scheduler
+#   Distros: openshift-enterprise
+#   Topics:
+#   - Name: Secondary scheduler overview
+#     File: index
+#   - Name: Secondary Scheduler Operator release notes
+#     File: nodes-secondary-scheduler-release-notes
+#   - Name: Scheduling pods using a secondary scheduler
+#     File: nodes-secondary-scheduler-configuring
+#   - Name: Uninstalling the Secondary Scheduler Operator
+#     File: nodes-secondary-scheduler-uninstalling
+# - Name: Using Jobs and DaemonSets
+#   Dir: jobs
+#   Topics:
+#   - Name: Running background tasks on nodes automatically with daemonsets
+#     File: nodes-pods-daemonsets
+#     Distros: openshift-rosa-hcp
+#   - Name: Running tasks in pods using jobs
+#     File: nodes-nodes-jobs
+# - Name: Working with nodes
+#   Dir: nodes
+#   Distros: openshift-rosa-hcp
+#   Topics:
+#   - Name: Viewing and listing the nodes in your cluster
+#     File: nodes-nodes-viewing
+#  cannot use oc adm cordon; cannot patch resource "machinesets"; cannot patch resource "nodes"
+#   - Name: Working with nodes
+#     File: nodes-nodes-working
+#  cannot create resource "kubeletconfigs", "schedulers", "machineconfigs", "kubeletconfigs"
+#   - Name: Managing nodes
+#     File: nodes-nodes-managing
+#  cannot create resource "kubeletconfigs"
+#   - Name: Managing graceful node shutdown
+#     File: nodes-nodes-graceful-shutdown
+#  cannot create resource "kubeletconfigs"
```

Comment:
@bhardesty Could you please confirm, what is the purpose for commenting these lines?

Reply:
These topics are originally from OCP. When we originally ported the OCP Nodes book to ROSA, the topics that were determined to be inapplicable to ROSA were commented out. Now for ROSA HCP, we've left them commented out on the basis that if they didn't apply to ROSA Classic, then they also won't apply to ROSA HCP.

Reply:
But we can create resource kubeletconfigs for HCP cluster

Reply:
I think this comment just means that you can't create a `KubeletConfig` CR as a dedicated-admin in a ROSA cluster. In looking at the procedure in OCP (https://docs.openshift.com/container-platform/4.17/nodes/pods/nodes-pods-plugins.html), I doubt that it would ever be applicable to ROSA, since it involves using device plugins whereas ROSA uses standard AWS instance types.

Reply:
Okay Thanks for the confirmation.